src/Microsoft.Health.Fhir.Core/Features/Operations/Export/GetExportRequestHandler.cs

```diff
@@ -38,7 +39,7 @@ public async Task<GetExportResponse> Handle(GetExportRequest request, Cancellati
                     outcome.JobRecord.QueuedTime,
                     outcome.JobRecord.RequestUri,
                     requiresAccessToken: false,
-                    outcome.JobRecord.Output,
+                    outcome.JobRecord.Output.Values.ToList(),
```

Comment:
it would be nice if this is ordered in some way (e.g., by resource type or eventually by resource type + part number).